server/controllers/splus.ts

```diff
@@ -6,10 +6,23 @@ import { TimetableRequest, TimetableMetadata, Timetable } from '../model/SplusEi
 import getEvents from '../lib/SplusApi';
 
 const CACHE_SECONDS = parseInt(process.env.SPLUS_CACHE_SECONDS || '10800');
+const AUTH_SECRET = process.env.AUTH_SECRET;
 
 const router = express.Router();
 const flatten = <T>(arr: T[][]) => [].concat(...arr) as T[];
 
+const isRequestFromOstfalia = (req) =>
+  (req.headers['x-forwarded-for'] || req.connection.remoteAddress).startsWith('141.41.');
+export const skedGuard = (req, res) => {
+  if (req.cookies['auth'] == AUTH_SECRET) return true;
```

Comment:
und wenn ich jetzt jemanden das Secret schicke ? :D

Reply:
Neue Änderung: Cookies werden signiert und laufen nach 14 Tagen ab, d. h. man muss sich in regelmäßigen Abständen in der Ostfalia einwählen. Ist das akzeptabel? 🤔

Reply:
das klingt schon besser
Meine Ostfalia Credentials will ich aber dafür trotzdem nicht nutzen :/